src/client/components/banner/banner.css

```diff
@@ -0,0 +1,20 @@
+.banner {
+  background-color: #dd1e1e;
+  color: #fff;
+  font-size: 115%;
+  font-weight: 900;
+  left: -8rem;
+  max-width: 31rem;
+  min-width: 31rem;
+  padding: 7rem;
+  position: fixed;
+  text-align: center;
+  top: 14rem;
+  -webkit-transform: rotate(-45deg);
```

Comment:
Is there a reason for specifying these vendor properties? I thought `transform` was pretty well supported by now, so using it should be fine?

Reply:
I believe that is what ChatGPT told me ;) I believe we can keep it. Doesn't hurt and maybe someone with a unsupported browser needs it?